### DISCUSSION TITLE
Exclude some templates from --incremental

### DISCUSSION THREAD

**Discussion by sentience** [11ty/eleventy#4130] [Enhancement Queue]
### Is your feature request related to a problem? Please describe.

My site is styled with TailwindCSS, which means whenever I change **/index.liquid** (for example), I need **/assets/styles/main.css** to be rebuilt by **eleventy-plugin-postcss** (via `addPassthroughCopy`).

Currently, when I make a change to **/index.liquid** with the Eleventy dev server running in `--serve --incremental` mode, **/assets/styles/main.css** does not get regenerated.

### Describe the solution you'd like

I'd like to be able to configure one of the following:

- Changes to ***.liquid** files (say) require a full rebuild.
- Changes to ***.liquid** files (say) require **/assets/styles/main.css** (per `addPassthroughCopy("assets")`) to be rebuilt.

### Describe alternatives you've considered

My current work-around is to run Eleventy dev server in non-incremental mode whenever I am making CSS changes to my site. I keep forgetting to do this, however, and being surprised when my styles aren't updating.

**Comment by zachleat** [comment]
First thought: I’m curious if the new `watch` property on the dev server helps?

https://www.11ty.dev/docs/dev-server/

Reminded of this post too https://www.webstoemp.com/blog/eleventy-dev-server-external-asset-pipeline/

**Comment by sentience** [comment]
Unfortunately adding `"**/*.liquid"` to the dev server's `watch` property doesn't change the behaviour of `--incremental`. Eleventy still only builds the Liquid template that was changed, not the main.css file that depends on it.

I take the point from that blog post that I'm likely better off doing my Tailwind CSS build with a separate dev server process, rather than attempting to run this from within Eleventy dev server. I'll aim to migrate in that direction.

**Comment by zachleat** [comment]
Sorry for the super late reply but I _think_ the `resetConfig` option of `eleventyConfig.addWatchTarget` might be _a_ path forward here? https://www.11ty.dev/docs/watch-serve/#reset-configuration

**Comment by dwkns** [comment]
You may also want to take a look at [eleventy-plugin-tailwind4](https://www.npmjs.com/package/eleventy-plugin-tailwindcss-4). This uses Eleventy's `before` hook to trigger PostCSS so runs every time a rebuild is detected. That should solve your problem. 

It also keeps everything within Eleventy so you can simplify your dev/build scripts in `package.json`
